lib/src/base-styles/_nx-colors.scss

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+:root {
+  --nx-color-border-default: #{$nx-indigo-100};
+  --nx-color-border-subsection: #{$nx-grey-200};
+  --nx-color-border-form-element: #{$nx-grey-1100};
+  --nx-color-border-form-element-pristine: #{$nx-indigo-100};
+  --nx-color-border-disabled: #{$nx-grey-500};
+  --nx-color-border-focus: #{$nx-teal-300};
```

Comment:
I think that these two variables should be grouped with the other interactive states below.

I'm wondering if we should name them something like `--nx-color-element-border-disabled` etc.

Reply:
> I think that these two variables should be grouped with the other interactive states below.
>
> I'm wondering if we should name them something like --nx-color-element-border-disabled etc.

I see nothing wrong with the way I've got them grouped.  All of the borders are together

Reply:
> > I think that these two variables should be grouped with the other interactive states below.
> > I'm wondering if we should name them something like --nx-color-element-border-disabled etc.
> 
> I see nothing wrong with the way I've got them grouped. All of the borders are together

Right I get that, but that's not the only way to group things. I'm suggesting that we group all of the interactive states together which I makes more sense in terms of how they're going to be used. For example the disabled border is almost always used with the disabled background, grouping them together makes it easier to see all of the interactive states in one glance rather than looking multiple places within the file.

Reply:
> Right I get that, but that's not the only way to group things. I'm suggesting that we group all of the interactive states together which I makes more sense in terms of how they're going to be used. For example the disabled border is almost always used with the disabled background, grouping them together makes it easier to see all of the interactive states in one glance rather than looking multiple places within the file.

Addressed in other thread: https://github.com/sonatype/sonatype-react-shared-components/pull/383#discussion_r635581410 7a3d082